databricks/koalas/tests/test_dataframe.py

```diff
@@ -569,3 +569,45 @@ def test_sample(self):
             kdf.sample()
         with self.assertRaises(NotImplementedError):
             kdf.sample(n=1)
+
+    def test_pivot_table(self):
```

Comment:
you should be able to replicate the dask test in https://github.com/dask/dask/blob/master/dask/dataframe/tests/test_reshape.py#L188

(without the categorical example)

Reply:
Thanks @thunterdb. 
I kept some tests as `Todo`, I'll update them once we implement hierarchical columns and column's name.

Reply:
Sounds great.

Reply:
Good.